spec/mobility/plugins/active_model/dirty_spec.rb

```diff
@@ -179,11 +183,7 @@ def save
         expect(instance.title_was).to eq("foo")
 
         Mobility.locale = :fr
-        if ENV['RAILS_VERSION'].present? && ENV['RAILS_VERSION'] < '5.0'
-          expect(instance.title_changed?).to eq(nil)
-        else
-          expect(instance.title_changed?).to eq(false)
-        end
+        expect(instance.title_changed?).to eq(false)
```

Comment:
The fact that `title_changed?` returned `nil` in AR 4.2 was actually a bug. Rather than mimic the bug, Mobility will now just return `false` as in later versions. This is technically a behaviour change but since 4.2 was never really fully supported, I don't intend to try to do anything about it.